salesagility/SuiteCRM
# Fixed #3068 - Cannot select with filter in prospect list

## Description



Fix for Issue [3068](https://github.com/salesagility/SuiteCRM/issues/3068) - Cannot select with filter in prospect list

Remove **double quote** in **current_query_by_page query** since it will not be accepted in securexsskey function (include/utils.php)

## How To Test This

1. First you create a prospect list.
2. Then you click on contacts, then a new windows appear.
3. Then you try to filter for some field.
4. And last you select all of them, and the result should be all the checked item must be added in contacts

## Types of changes


### Final checklist
